### PR TITLE
send UserActivityLogger events when tablet is opened or closed

### PR DESCRIPTION
**Pull request by sethalves**
- send UserActivityLogger events when tablet is opened or closed

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- no user-visible changes

**Comment by hifi-gustavo** [comment]
Build console output for commit: e3af5032c0281e46dc25020525cb49c4905800a2 available at: https://gist.github.com/0d9ecfd830ad625f4243fdbdfa11e510<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9621-e3af503.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9621-e3af503.dmg

**Comment by claudioshg** [comment]
testing

**Comment by cozza13** [comment]
Tested and confirmed with Metabase the data was there.

File: scripts/system/tablet-ui/tabletUI.js [hunk 1]
```diff
@@ -86,7 +88,6 @@
     var accumulatedLevel = 0.0;
     // Note: Might have to tweak the following two based on the rate we're getting the data
     var AVERAGING_RATIO = 0.05;
-    var MIC_LEVEL_UPDATE_INTERVAL_MS = 100;
```
Comment (PhilipRosedale) [inline scripts/system/tablet-ui/tabletUI.js]: is this variable no longer used and just needed to be deleted?  Doesn't seem related to this PR.

Comment (davidkelly) [inline scripts/system/tablet-ui/tabletUI.js]: looks like it isn't used, and the mic is updated when updateShowTablet is called.  That setInterval has hard-coded 100 for the interval.  Maybe worth having a constant (like the one removed) for that.
